src/components/ProfileView.jsx

```diff
@@ -21,8 +24,9 @@ import { AUTH_MODE } from 'actions/twoFactor'
 
 const PasswordSection = () => {
   const { t } = useI18n()
-
-  return (
+  const client = useClient()
+  const canAuthWithPassword = get(client, 'capabilities.can_auth_with_password')
```

Comment:
capabalities.can_auth... is injected by the Stack in cozy-data, right? So I imagine that we do : `new CozyClient(..., { capabilities : { can_auth...`, but where do we get access to this property from client itself? Shouldn't be get(client.options, 'capabilities'...) ?

Reply:
I had not yet opened this PR : https://github.com/cozy/cozy-client/pull/904 ;)